app/components/notification-list-row/template.hbs

```diff
@@ -0,0 +1,19 @@
+{{#if updateFrequency.isRunning}}
```

Comment:
While `updateFrequency` is running you don't need to show a skeleton. It would probably make the most sense to just add `.enqueue()` to the task (see [docs](http://ember-concurrency.com/docs/task-concurrency) for details) so that if the user changes their selection multiple times the tasks won't interfere with one another.

Reply:
Since you don't need the skeleton here you can just put the skeleton in the `notification-list` template instead of having a component.